src/components/show_card/ShowCard.js

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+import './ShowCard.css'
+
+const ShowCard = ({show}) => {
```

Comment:
This type of destructuring is something that we have to get into the habit of doing all the time, so great implementation.